src/AmbientSounds.Uwp/App.xaml.cs

```diff
@@ -565,4 +474,100 @@ private static IServiceProvider ConfigureServices(IAppSettings? appsettings = nu
 
         return provider;
     }
+
+    /// <summary>
+    /// Configures services used by the app.
+    /// </summary>
+    /// <param name="services">The target <see cref="IServiceCollection"/> instance to register services with.</param>
+    [Singleton(typeof(HttpClient))]
+    [Singleton(typeof(SoundListViewModel))] // shared in main and compact pages
+    [Transient(typeof(ScreensaverViewModel))]
+    [Singleton(typeof(ScreensaverPageViewModel))]
```

Comment:
@dpaulino the comment in the previous code said that "all viewmodels have to be transient unless otherwise specified", but almost all viewmodels are actually declared as singletons. I've kept that as is, but like... Was that intentional? 🤔

Reply:
I think that was an old comment that can be ignored